pyproject.toml

```
@@ -28,7 +28,7 @@ classifiers = [
 ]
 dependencies = [
     "jupyter_server>=2.0.0,<3.0.0",
-    "jupyter_ydoc>=2.0.0,<3.0.0",
+    "jupyter_ydoc>=2.1.2,<3.0.0",
     "pycrdt",
     "pycrdt-websocket>=0.14.0,<0.15.0",
```

Don't we need to update `pycrdt-websocket` too?
```suggestion
    "pycrdt-websocket>=0.15.0,<0.16.0",
```

---

Good catch, this is the matter of the PR :smile: